### ISSUE TITLE
Han dejado de funcionar todo

### ISSUE THREAD

**Issue by amaciasf** [tvalacarta/tvalacarta#51]
Buenas! El plugin ha dejado de mostrar toda la programación en todos los canales desde hace un par de días. He desinstalado  e instalado varias veces la versión 4.2.0 y sigue igual.

Muchas gracias, gran trabajo!!

**Comment by tvalacarta** [comment]
A qué te refieres exactamente?

He probado a entrar y no he visto nada raro... tanto la sección "Programas" como la sección "Canales" funcionan bien.

**Comment by amaciasf** [comment]
Tenía que solo me pasará a mi, ya que me parecía raro que nadie hubiese dicho nada...

Por ejemplo, una vez dentro del plugin me meto en Canales / Todos / Dplay y sale completamente vacio exceptuando el icono para volver atrás. Y esto me pasa con todos los canales.

Estoy bajo LibreELEC 9.2.0 pero si dices que te va será algo particular de mi instalación. Investigaré más.

Muchas gracias por todo, saludos!

**Comment by tvalacarta** [comment]
En el log saldrán más detalles, pero por lo que dices parece que Kodi no puede conectar a Internet. Prueba a instalar algún add-on del repositorio oficial de Kodi, a usar el add-on de YouTube o algo así, a ver qué resultado obtienes.

**Comment by amaciasf** [comment]
Sí, lo probé con Youtube y funciona bien. He puesto LibreELEC en modo a prueba de errores, lo instalo y ahí sí funciona.

Mañana restauro LibreELEC a cero a ver qué tal.

Gracias por todo, saludos!
